view/frontend/web/js/action/express-pay/update-quote-wallet-pay-action.js

```diff
@@ -0,0 +1,40 @@
+define(
+    [
+        'uiRegistry',
+        'jquery',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/action/place-order',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Bold_CheckoutPaymentBooster/js/action/express-pay/get-express-pay-order-action',
+        'Bold_CheckoutPaymentBooster/js/action/express-pay/update-quote-address-action',
+    ],
+    function (
+        registry,
+        $,
+        quote,
+        placeOrderAction,
+        redirectOnSuccessAction,
```

Comment:
Looks like some of these dependencies aren't used in this file. might only need the `updateQuoteAddressAction` if i'm not mistaken. 

Otherwise everything else looks great!

Reply:
Thanks for pointing this out. I've removed the unused dependencies.